### ISSUE TITLE
Other localizations wont be saved

### ISSUE THREAD

**Issue by AlexMl** [msummers123/Movecraft-3#122]
I am not sure if this is an issue but it seems to me that the localizations you have under src/main/resources/localisation will not be saved (except from the English language). Also it wont load the files from the resources if i set the locale in the config file. So the translated files are actually useless.

Is it designed that way or did i just missed something here?

**Comment by msummers123** [comment]
I'll be honest, I'm not sure how those resources are saved. I haven't really worked with it. I can look into it when I can, although I'm about to head out of the country for a few weeks and I won't have internet. So it may be a while.

**Comment by AlexMl** [comment]
No problem. I can do something here to contribute that part. I'll fork it and come back with a pull request

**Comment by mwkaicz** [comment]
Yes, @AlexMl is right, localisation are not saved, but nobody missing it :) I think we are only one CZ server with Movecraft and majority uses EN version ...who wants own language, he just overwrite/copy EN file :)

**Comment by AlexMl** [comment]
You are right. I just thought it would be nice to copy the things from the plugin in a file in the localization folder to provide a default set of  translations. But I see its not really necessary.

**Comment by mwkaicz** [comment]
I have some uncommited changes now, I'll fix it.
